packages/jest-mock/src/__tests__/index.test.ts

```diff
@@ -542,6 +542,29 @@ describe('moduleMocker', () => {
       });
     });
 
+    it('mocks the method in the passed object itself', () => {
+      const parent = {func: () => 'abcd'};
+      const child = Object.create(parent);
+
+      moduleMocker.spyOn(child, 'func').mockReturnValue('efgh');
+
+      expect(child.hasOwnProperty('func')).toBe(true);
+      expect(child.func()).toEqual('efgh');
+      expect(parent.func()).toEqual('abcd');
+    });
+
+    it('should delete previously inexistent methods when restoring', () => {
+      const parent = {func: () => 'abcd'};
+      const child = Object.create(parent);
+
+      moduleMocker.spyOn(child, 'func').mockReturnValue('efgh');
+      moduleMocker.restoreAllMocks();
+      moduleMocker.spyOn(parent, 'func').mockReturnValue('jklm');
```

Comment:
Can we check that it's `abcd` before spying again just to be sure that restore doesn't lose the original completely

Reply:
Yes, that's a great idea!

And btw thanks for the review and the positive feedback ✨ 

Looking forward to being around here more often 💖